styles.css

```diff
@@ -0,0 +1,79 @@
+div.container {
+  width: 60rem;
+  height: 640px;
+}
+
+.row {
+  display: flexboxgrid;
+  flex-direction: column;
+}
+
+.col {
+  flex: 1;
+  margin: 0.1em;
+  padding: 0.8em;
+}
+
+div.difference-header {
```

Comment:
You can omit the `div` part of this selector. `.difference-header` should suffice.